src/Test/VarnishTestCase.php

```diff
@@ -11,152 +11,12 @@
 
 namespace FOS\HttpCache\Test;
 
-use FOS\HttpCache\ProxyClient\Varnish;
-use FOS\HttpCache\Test\Proxy\VarnishProxy;
-
 /**
- * A phpunit base class to write functional tests with varnish.
- *
- * You can define constants in your phpunit to control how this test behaves.
- *
- * Note that the WEB_SERVER_HOSTNAME must also match with what you have in your
- * .vcl file.
- *
- * To define constants in the phpunit file, use this syntax:
- * <php>
- *     <const name="VARNISH_FILE" value="./tests/Functional/Fixtures/varnish-3/fos.vcl" />
- * </php>
- *
- * VARNISH_FILE         Varnish configuration file (required if not passed to setUp)
- * VARNISH_BINARY       executable for Varnish. this can also be the full path
- *                      to the file if the binary is not automatically found
- *                      (default varnishd)
- * VARNISH_PORT         port Varnish listens on (default 6181)
- * VARNISH_MGMT_PORT    Varnish management port (default 6182)
- * VARNISH_CACHE_DIR    directory to use for cache
- *                      (default sys_get_temp_dir() + '/foshttpcache-varnish')
- * VARNISH_VERSION      Version of varnish: 3 or 4, defaults to 4.
- * WEB_SERVER_HOSTNAME  hostname/IP your application can be reached at (required)
+ * Abstract test that contains traits necessary for running tests against Varnish
  */
-abstract class VarnishTestCase extends ProxyTestCase
+abstract class VarnishTestCase
```

Comment:
well, if it does not extend the base testcase class, it won't run as it won't be a PHPUnit testcase.

https://travis-ci.org/FriendsOfSymfony/FOSHttpCache/jobs/74553567 was running 78 tests while https://travis-ci.org/FriendsOfSymfony/FOSHttpCache/jobs/74556258 ran 112 tests